builder/build.go

```diff
@@ -93,11 +93,15 @@ func CopyFiles(src string, destination string, recursive bool) {
 		log.Fatal(err)
 	}
 
+	src = strings.TrimRight(src, "/") + "/"
+	destination = strings.TrimRight(destination, "/") + "/"
+
 	for _, file := range files {
 
 		if file.IsDir() == false {
-
-			cp(src+"/"+file.Name(), destination+file.Name())
+			if err := cp(src+file.Name(), destination+file.Name()); err != nil {
+				log.Fatal(err.Error())
```

Comment:
This feels quite final.. what if we collected the errors in a slice and then printed them all at the top level of this function?